src/nmodl/noccout.cpp

```diff
@@ -922,12 +921,10 @@ diag("current can only be LOCAL in a BREAKPOINT if CONDUCTANCE statements are us
 	P("  if (!_first) return;\n");
 	printlist(initlist);
 	P("_first = 0;\n}\n");
-	P("\n#if defined(__cplusplus)\n} /* extern \"C\" */\n#endif\n");
+	//P("\n#if defined(__cplusplus)\n} /* extern \"C\" */\n#endif\n"); // TODO - can this be removed?
```

Comment:
without looking into generated code - this is printed in `.c` file generated by nmodl. So I think it should be still there in principle?

Reply:
Well we are still generating `.c` so it is not needed. But will put it back though, it is a remnant of `extern C` removal exploration in the beginnings of the work.